.github/workflows/components-release.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and Publish Components Package
+
+on:
+  release:
+    types: [published]    
+  workflow_dispatch:
+    inputs:
```

Comment:
While dispatching, if one of the steps here isn't dynamically identifying the latest release tag, we might want to require the release tag as input here. 

You might like to grab the release tag lookup "function" from another one of the makefiles / workflows.